docs/rules/func-name-matching.md

```diff
@@ -97,6 +85,50 @@ module.exports = function foo(name) {};
 module['exports'] = function foo(name) {};
 ```
 
+## Options
+
+This rule takes an optional string of "always" or "never" (when omitted, it defaults to "always"), and an optional options object with two properties `considerPropertyDescriptor` and `includeCommonJSModuleExports`.
+
+### considerPropertyDescriptor
+
+A boolean value that defaults to `false`. If `considerPropertyDescriptor` is set to true, the check will take into account the use `Object.create`, `Object.defineProperty`, and `Object.defineProperties`.
```

Comment:
This is missing `Reflect.defineProperty`

Reply:
It's better to be included. I'll update PR soon!